packages/runtime-vapor/src/dom/prop.ts

```diff
@@ -242,45 +243,10 @@ function shouldSetAsProp(
     return false
   }
 
-  // these are enumerated attrs, however their corresponding DOM properties
-  // are actually booleans - this leads to setting it with a string "false"
-  // value leading it to be coerced to `true`, so we need to always treat
-  // them as attributes.
-  // Note that `contentEditable` doesn't have this problem: its DOM
-  // property is also enumerated string values.
-  if (key === 'spellcheck' || key === 'draggable' || key === 'translate') {
+  if (shouldSetAsAttr(el.tagName, key)) {
```

Comment:
```typescript
if(cache[`${tag}_${key}`] ?? shouldSetAsAttr(el.tagName, key)) {...}
```

this can avoid lots of function invoke which I think may be helpful for performance

Reply:
done via https://github.com/vuejs/vue-vapor/pull/291/commits/26cfbb191a845361f8a775695aacda9cf7ce7bc5